provisioning/miniprov/hare_mp/main.py

```diff
@@ -1090,14 +1092,6 @@ def create_parser() -> argparse.ArgumentParser:
              'for no size limit.',
         action='store')
 
-    sb_sub_parser.add_argument(
```

Comment:
where are we accepting -c <config url> and -s <services> args?

Reply:
Here we are adding -c and -s args:
> 
         parser.add_argument('--config', '-c',
                        help='Conf Store URL with cluster info',
                        required=config_required,
                        nargs=1,
                        type=str,
                        action='store')

>
     def add_service_argument(parser):
          parser.add_argument('--services', '-s',
                        help='Services to be setup. Supported '
                        'values: ' + str([s.value for s in Svc]),
                        nargs=1,
                        type=Svc,
                        action='store')

Reply:
ohh ok. I was not able to see those args in the args.
so wanted to confirm.